### PR TITLE
`[scheduling]` Add a scheduling tab.

### PR DESCRIPTION
**Pull request by charlesg3**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
image

### CODE REVIEW

File: frontend/src/pages/Clusters/ClusterScheduling.js [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    const headNode = getState([...clusterPath, 'headNode']);
+    setState(pendingPath, true);
+    CancelJob(headNode.instanceId, user, jobId, () => refreshQueues(() => clearState(pendingPath)))
+  }
+
+  return (
+    <div>
+      {job.job_state !== "COMPLETED" &&
+      <div>
+        <Button loading={pending} onClick={() => {cancelJob(job.job_id)}}>Stop</Button>
```
Comment (sean-smith) [inline frontend/src/pages/Clusters/ClusterScheduling.js:83]: Button should be named `Cancel`. `Stop` raises the idea of being able to start the job back up

File: api/PclusterApiHandler.py [hunk 2]
```diff
@@ -181,6 +181,78 @@ def get_cluster_config():
...
+    parser.add_argument("region", type=str)
+    args = parser.parse_args()
+    user = args.get("user", "ec2-user")
+    instance_id = args.get("instance_id")
+
+    jobs = ssm_command(
+        args.get("region"),
+        instance_id,
+        user,
+        "squeue --json | jq .jobs\\|\\map\\({name,nodes,partition,job_state,job_id\\}\\)",
```
Comment (sean-smith) [inline api/PclusterApiHandler.py:236]: Can we include runtime also? This is important for long running zombie jobs that may need to be killed.

```
squeue --json | jq .jobs\\|\\map\\({name,nodes,partition,job_state,job_id,time\\}\\)
```

Comment (charlesg3) [reply api/PclusterApiHandler.py]: done

File: frontend/src/pages/Clusters/ClusterScheduling.js (hunk 1 above)
Comment (charlesg3) [reply frontend/src/pages/Clusters/ClusterScheduling.js]: done.

**Pull request merged**